enhancements/sig-architecture/30-clusterset-override/README.md

```diff
@@ -0,0 +1,191 @@
+# Adding Exclusive Key in ManagedClusterSet
+ 
+## Release Signoff Checklist
+ 
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [website](https://github.com/open-cluster-management/website/)
+ 
+## Summary
+The proposed work enhances the managedClusterSet API to support managedClusterSet Override.
+ 
+## Motivation
+Today, managedClusterSet is designed to be exclusive. A managedCluster can only be included in only one managedClusterSet, by setting a label `cluster.open-cluster-management.io/clusterset` on the managedCluster. A user must have explicit RBAC rules, `managedclustersets/join`, to add such labels.
```

Comment:
Can you add a section to the proposal that explains why the exclusivity was originally a design goal, why we are changing that behavior, and how we expect this to impact clients.

Reply:
done